.github/workflows/contributors_update.yml

```diff
@@ -0,0 +1,74 @@
+name: Update Contributors in README
+
+on:
+  push:
+    branches:
+      - main
+  schedule:
```

Comment:
we can remove this schedule part.

Contributors will change only during the push (if any new) so need to run this workflow after every specific time period